Index.html

```diff
@@ -1,231 +1,328 @@
+						/>
+					</a>
+					<h3 class="multiPost">Multi-Post Stories</h3>
+					<p class="projectBrief">
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required. has been the industry's standard dummy text ever
+						since the 1500s, when an unknown printer took a standard dummy text.
+					</p>
+					<div class="programmingLanguages">
+						<ul class="languagesUsed">
+							<li>css</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>html</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>bootstrap</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>Ruby</li>
+						</ul>
+					</div>
+					<button class="projectButton" type="Button">See Project</button>
+				</div>
+				<div id="project3">
+					<a href="#">
+						<img
+							src="./assets/Img-Placeholder.svg"
+							alt="Screenshot of Project"
+						/>
+					</a>
+					<h3 class="multiPost">Multi-Post Stories</h3>
+					<p class="projectBrief">
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required. has been the industry's standard dummy text ever
+						since the 1500s, when an unknown printer took a standard dummy text.
+					</p>
+					<div class="programmingLanguages">
+						<ul class="languagesUsed">
+							<li>css</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>html</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>bootstrap</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>Ruby</li>
+						</ul>
+					</div>
+					<button class="projectButton" type="Button">See Project</button>
+				</div>
+				<div id="project4">
+					<a href="#">
+						<img
+							src="./assets/Img-Placeholder.svg"
+							alt="Screenshot of Project"
+						/>
+					</a>
+					<h3 class="multiPost">Multi-Post Stories</h3>
+					<p class="projectBrief">
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required. has been the industry's standard dummy text ever
+						since the 1500s, when an unknown printer took a standard dummy text.
+					</p>
+					<div class="programmingLanguages">
+						<ul class="languagesUsed">
+							<li>css</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>html</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>bootstrap</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>Ruby</li>
+						</ul>
+					</div>
+					<button class="projectButton" type="Button">See Project</button>
+				</div>
+			</div>
+			<img class="line" src="./assets/vectorLine.svg" alt="horizontal line" />
+		</section>
+		<footer class="contact">
+			<div class="contact-text">
+				<h2>CONTACT US</h2>
+				<p>
```

Comment:
Hi @BenjaminSemah, let's give these two a class attribute and style them so they can have the same font as the rest of the portfolio as specified by the previous reviewer. Your paragraph is hard to read.
Your design
![con](https://user-images.githubusercontent.com/53356820/144106959-f284c158-9cee-40be-88fd-04d8230493c2.png)
Figma design
![con1](https://user-images.githubusercontent.com/53356820/144107003-fb42f8cb-10e5-43f5-a1a0-85c9ab61459e.png)

Reply:
Hello, @keneogu thanks for your review. 

However, I'll want to point out that I implemented **ALL** of the changes requested by the previous reviewer and not "some" as you stated in your comment.

Also, the h2 and p elements in the screenshot you posted uses different fonts ("Roboto" for the h2 element and "Inter" for the p element) and different styles. So it seems using a class attribute here to style both of them won't be ideal.

The CSS I wrote (posted in the screenshot below) is consistent with the style guide in Figma. So I would be glad if you could take a second and give me feedback so I can understand what you want me to do.

**CSS in my project and style guide in Figma for the h2**
![Screenshot (253)](https://user-images.githubusercontent.com/88531553/144115529-a24daf83-c3a7-4840-b433-3d050ffbe832.png)

![Screenshot (255)](https://user-images.githubusercontent.com/88531553/144115684-6ac42249-c0b3-498b-9971-5de618058953.png)

**CSS in my project and style guide in Figma for the paragraph.**
![Screenshot (252)](https://user-images.githubusercontent.com/88531553/144116109-92847763-7bf1-4573-87eb-5620f279ef56.png)

![Screenshot (257)](https://user-images.githubusercontent.com/88531553/144116271-8f6ed62c-4694-43c6-a066-fb92f5b1c46f.png)

Thanks.